docs/docusaurus/docs/core/define_expectations/create_an_expectation.md

```diff
@@ -0,0 +1,178 @@
+
+   In your code, you will find the classes for Expectations in the `expectations` module:
+
+   ```python title="Python"
+   from great_expectations import expectations as gxe
+   ```
+
+2. Determine the Expectation's evaluation parameters
+
+   To determine the parameters your Expectation uses to evaluate data, reference the Expectation's entry in the [Expectation Gallery](https://greatexpectations.io/expectations/).  Under the **Args** section you will find a list of parameters that are necessary for the Expectation to be evaluated, along with the a description of the value that should be provided.
+
+   Parameters with a default value do not need to be set.  Parameters that are set to `None` can be provided at runtime.  Values for all other parameters must be provided when the Expectation is created.
+
+   For example, take the [**Args** section for `ExpectColumnMaxToBeBetween` in the Expectation Gallery](https://greatexpectations.io/expectations/expect_column_max_to_be_between#args).  In this section, you will see that for `ExpectColumnMaxToBeBetween`'s parameters:
+
+      - `column` must be a string.  Since it cannot be set to `None` its value must be provided when the Expectation is created.
+      - `min_value` and `max_value` can be either set when the Expectation is created, or be set to `None` when the Expectation is created and then have appropriate values passed in at runtime, instead.
+      - `strict_min` and `strict_max` both have default values that can be overriden when the Expectation is created, or left as is.  Because they cannot be set to `None` when the Expectation is created these parameters also cannot be passed in at runtime.
+
+3. Optional. Determine the Expectation's other parameters
+
+   In addition to the parameters that are required for an Expectation to evaluate data all Expectations also support some standard parameters that determine how strictly Expectations are evaluated and permit the addition of metadata.  In the Expectations Gallery these are found under each Expectation's **Other Parameters** section.
+
+   These parameters are:
+
+   <StandardArgumentsTable/>
+
+4. Create the Expectation.
+
+   Using the Expectation class you picked and the parameters you determined when referencing the Expectation Gallery, you can create your Expectation.
+
+   In this example the `ExpectColumnMaxToBeBetween` Expectation is created and all of its parameters are defined in advance while leaving `strict_min` and `strict_max` as their default values:
+
+   ```python title="Python"
+   preset_expectation = gxe.ExpectColumnMaxToBeBetween(
+       column="passenger_count",
+       min_value=4,
+       max_value=6
+   )
+   ```
+   
+   While in this example the same `ExpectColumnMaxToBeBetween` Expectation is created, but the values for `min_value` and `max_value` will be provided at runtime rather than set when the Expectation is created:
```

Comment:
```suggestion
   While in this example the same `ExpectColumnMaxToBeBetween` Expectation is created, the values for `min_value` and `max_value` will be provided at runtime rather than set when the Expectation is created:
```